### ISSUE TITLE
Bootstrap msm is slow 

### ISSUE THREAD

**Issue by msultan** [msmbuilder/msmbuilder#745]
The code is annoyingly slow. I think what is happening is that solving the eigensystem on the individual processors is holding things up whereas something like the ```bayesmsm``` is returning the tmats which are then processed sequentially on the fly. 

Two possible solutions:
1). Forget the parallel and fit the bootstrap models sequentially. 
2). Keep the parallel and only return the tmat ala ```bayesmsm``` and calculate everything sequentially . 

Basically, I need to un-engineer this because I thought I would be clever with parallelization.
